apps/admin-x-activitypub/src/components/feed/ArticleModal.tsx

```
@@ -738,289 +751,297 @@ const ArticleModal: React.FC<ArticleModalProps> = ({
+                                                onItemClick={scrollToHeading}
+                                            />
+                                        </div>
+                                    </div>
+                                )}
+                                <div className='grow overflow-y-auto'>
+                                    <div className={`mx-auto px-8 pb-10 pt-5`} style={{maxWidth: currentMaxWidth}}>
+                                        {threadParents.map((item) => {
+                                            return (
+                                                item.object.type === 'Tombstone' ? (
+                                                    <DeletedFeedItem last={false} />
+                                                ) : (
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)